src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -87,20 +91,41 @@
     dataByDate = groupDataByYear(flattenData(data));
   }
 
+  function removeTags(html) {
+    return html.replace(/<\/?[^>]+(>|$)/g, "");
+  }
+
   function getNotificationText() {
     if ($boundary.id === "locagrid") {
       return "";
     } else {
-      return notificationText.replace(/<\/?[^>]+(>|$)/g, "");
+      return removeTags(notificationText);
+    }
+  }
+
+  function getUncertaintyText() {
+    if (!$uncertaintyStore || $indicator.id !== "intensity") return "";
+    const { lowSampleSize, nullCIValues } = $uncertaintyStore;
+    if (lowSampleSize && nullCIValues) {
+      return removeTags(`${warningLowSampleSize}. ${warningMissingCI}`);
```

Comment:
I don't think you need the period here between the two `${}`

Reply:
good catch. fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/206/commits/609e284c54149dcec8b3d2aed9f4f5044ab9bcf6